ntbs-integration-tests/HomePage/HomePageTests.cs

```diff
@@ -32,5 +32,44 @@ public async Task DismissAlert_CorrectlyDismissesAlertAndReturnsHomePage()
 
             Assert.Null(reloadedPageContent.QuerySelector("#alert-1"));
         }
+
+        [Fact]
+        public async Task ShowingHomepageKpis_WhenUserIsNationalUser()
+        {
+            // Arrange

```

Comment:
Though it is the default client, I think it is worth dropping in a local 'client' here too, just to conform to the same patterna s the other tests.